BrunoEmaSN/url-shortener
# Short 01: Create Login and register flow

## Description
The flow needs to be created to log in and register to the system.
At the same time, it is required that the dashboard, docs and settings routes be protected so that they can only be accessed using a token.

## Task
[short-1](https://www.notion.so/Create-Login-and-register-flow-48d09338aa14409a842a49f11246632a?pvs=4)

## How to test
1. Add the following variable NEXT_PUBLIC_ BASE_URL to dotenv
2. go to register (/auth/register)
3. test the fields (name, email, password and confirm password)
4. create account
5. go to login (/auth/login)
6. test the fields (email and password)
7. login
8. test protected routes [dashboard, docs, settings]
